### DISCUSSION TITLE
First impressions on argon pro

### DISCUSSION THREAD

**Discussion by DavidHusicka** [ppy/osu#21841] [Ideas]
As a foreword, I'd like to say that these are my personal opinions on argon pro and if you disagree with any opinion, you can disregard them as skinning is very opinionated topic.

I find argon too distracting for following reason:

## The flashes after clicking circles are too strong and distracting

I find these flashed too strong. They are at times even brighter than the hitcircles themselves. That's not really good for readability in my opinion. Hitcircles can fade out but flashing is just too much of a distractions for me personally. Especially if these flashes can obscure other elements.

(argon pro)
![image](https://user-images.githubusercontent.com/25853767/209716013-9aac3721-477f-40b2-af1f-36a4a601abe9.png)
(my current skin)
![image](https://user-images.githubusercontent.com/25853767/209716039-aedfcd77-fa1d-4cb5-b941-f5bd99bda52d.png)

## Reverse arrow is too small
 
I personally find the reverse arrow too small to notice. Combining it with overall crowdness of objects on the screen on argon doesn't make it a good combination. My personal preference is to have a big half-circle instead of an arrow.

(argon pro)
![image](https://user-images.githubusercontent.com/25853767/209716708-c088f231-19c5-4b09-b1db-aecfe99f4701.png)
(my current skin)
![image](https://user-images.githubusercontent.com/25853767/209716743-d786bfd2-cebc-4605-8e9e-3b5ae3edeaf7.png)

## Cursor is hard to see

Since hitcombos are bright colors, the cursor doesn't have necessary contrast and gets lost at times. 

See pictures above.

## Color changes between combos are too big

Since combo colors use bright and vivid colors, it's just distracting to me on PC. That's all I can say.

## Numbers on circles

Again, this is just a personal preference but I find these numbers distracting.  If I need to know the order, I just look at these arrows between circles and know which one's next. I'd personally make the arrows on argon stronger. I don't think that these numbers should be removed because there's going to be a lot of people who are finding them genuinely helpful.

# Would dimming the colors help?

It would solve some of them partially but it would introduce other issues. Especially on mobile devices. See https://github.com/ppy/osu/issues/6356 (tl;dr, I argued about the exact opposite some time ago since bright colors are really good on mobile)

# Positives

I like the idea of a built-in pro skin. I like the minimalistic design but it could be pushed a little further. I also dig these hitsounds. They are great.

Again these are my personal opinions. Feel free to disagree.

**Comment by ar1a** [comment]
Adding onto this, the followpoints are basically useless being so small and on screen for so long, and the afterimage of hitting a note is very bright and distracting, i've made *many* misreads trying to hit an afterimage in the middle of 1-2 jumps or something. Is there no way for us to tweak the argon skin? is it all hidden behind recompiling the client?

**Reply by mk56-spn** [reply]
It does indeed need a recompile in its current state to have any changes applied, since it's pretty much entirely hard coded into the game at this point in time

**Comment by mcpower** [comment]
Regarding the "flashes", I find that they block the visibility of less-than-maximum judgements, especially during a stream:

![osu_2022-12-28_20-39-51](https://user-images.githubusercontent.com/6015058/209797615-b81c086b-9200-4966-9232-914e2adfb935.png)

It's hard to spot the "OKs" in this image, but there are actually four of them! Some skins show this better than others - for example, the BTMC "Freedom Dive" skin shows three with one hidden under the cursor:

![osu_2022-12-28_20-38-49](https://user-images.githubusercontent.com/6015058/209797930-f5b1480b-7b80-4e6a-99f8-cdffc9dfc237.png)

while the "osu!celebrity" skin shows all four, as its 100 sprite is vertically offset and large:

![osu_2022-12-28_20-39-17](https://user-images.githubusercontent.com/6015058/209798184-68aeb99b-4354-4c98-a6f9-8d1b97677a94.png)

**Comment by Speykious** [comment]
> Adding onto this, the followpoints are basically useless being so small and on screen for so long

I agree. They're so hard to read that I basically don't try to read them.
![image](https://user-images.githubusercontent.com/13885008/209818686-64d5d07f-3c46-40b0-87d1-7cf5f767e027.png)

Instead I would opt for very straight lines like in this image of [a random skin I found](https://osuskins.net/skin/q1nERg3) (there are lots of skins with these kinds of followpoints).
I'm sure there's a way for straight line followpoints to fit with the argon skin design :D
![image](https://user-images.githubusercontent.com/13885008/209819271-8915d4a0-4a34-46de-8912-72ffdf13b245.png)

**Reply by yamplum** [reply]
To better illustrate the issue, here's what the same pattern looks like with current argon follow points vs straight lines:

![osu!_h4gNc6nWwu](https://user-images.githubusercontent.com/118788705/210285488-5c2ea809-03a8-4a9f-9934-b99f64b5dd64.png)
![osu!_nWGFpYGrJo](https://user-images.githubusercontent.com/118788705/210285491-a4f07210-d8c0-4eda-9ea1-6c7beb86aebc.png)

**Comment by ghost** [comment]
I find that the white outlines on the skin distracting. It appears to extend outside of the circle, where any other pro skin has the white border completely within the bounds of it (you can see the outline is bigger than the sliderbody). Also, the sliderbodies are completely opaque, letting circles get hidden underneath them.

> Again, this is just a personal preference but I find these numbers distracting. If I need to know the order, I just look at these arrows between circles and know which one's next. I'd personally make the arrows on argon stronger. I don't think that these numbers should be removed because there's going to be a lot of people who are finding them genuinely helpful.

Perhaps this can be an option like hit lighting.

**Reply by GopoBoi** [reply]
the white outline is actually part of the circle. sliderbodies are smaller.

**Comment by peppy** [comment]
The pro skin, as mentioned everywhere, only has hitsounds and greats hidden. Nothing else is changed. While you're welcome to give feedback, most of what "that pro audience wants" is already well known to us.

**Reply by yamplum** [reply]
Sorry, just to clarify: does this mean that more changes will be coming down the line, or that argon pro is not planned to significantly diverge from argon?

**Reply by peppy** [reply]
Probably more changes, or it will be replaced with proper skin settings / presets (end goal).
